nistats/second_level_model.py

```diff
@@ -406,8 +410,7 @@ def compute_contrast(
         # Fit an OLS regression for parametric statistics
         Y = self.masker_.transform(effect_maps)
         if self.memory is not None:
-            arg_ignore = ['n_jobs']
-            mem_glm = self.memory.cache(run_glm, ignore=arg_ignore)
+            mem_glm = self.memory.cache(run_glm, ignore=['n_jobs'])
```

Comment:
You can simply write
```
if self.memory:
```

Reply:
I agree, I also make the same edit in https://github.com/nistats/nistats/blob/master/nistats/first_level_model.py